src/worldmap_ctrl.ts

```diff
@@ -35,6 +36,8 @@ const panelDefaults = {
   showLegend: true,
   mouseWheelZoom: false,
   hideLocationName: false,
+  isLogScale: false,
+  logFunction: 'log10',
```

Comment:
Minor, but if the `isLogScale` is `false` by default then `logFunction` could be `undefined` by default (as well), if my understanding is correct.

Reply:
I understand but I don't wanna do that. Why: Because, when you check the isLogScale, if you don't change the input, the log function would be undefined (requiring the user to actively choose a function on the selectbox). This way, the selectbox will display the default one, shown here and, if the user wants to change, than he can.
Otherwise I would need to check if it is set, use a default if not and, also, make sure the select shows the default if none selected. 10 lines of code when the solution above did it.